cdap-hbase-compat-0.98/pom.xml

```diff
@@ -57,6 +57,17 @@
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-tms</artifactId>
```

Comment:
So we need this dependency on every compat module? What classes are actually needed from `cdap-tms`?